cache_images/ubuntu_packaging.sh

```diff
@@ -118,6 +118,7 @@ INSTALL_PACKAGES=(\
     zlib1g-dev
 )
 DOWNLOAD_PACKAGES=(\
+    crun
```

Comment:
Why DOWNLOAD_PACKAGES? Why not in INSTALL_PACKAGES?

Reply:
@mheon wanted this so he could test some things.  I didn't want to risk mucking up everybody's tests by having both `runc` and `crun` installed at the same time.  Am I wrong, is that a perfectly safe thing to do?

Reply:
more context: None of the Ubuntu VMs have cgv2 enabled by default on the kernel.  @mheon is expecting this.

Reply:
I have to assume that podman will figure out whether to use runc or crun; but this has to be decided by @mheon.

@cevich, just so @mheon (and I) can understand the full ramifications, can you please ELI5 the difference between DOWNLOAD and INSTALL? I think I have a small understanding of it but don't want to misspeak.

Reply:
I don't actually know what the default runtime chosen will be on a cgroupsv1 system with runc and crun both installed, I'd need to look at the code to check...

Reply:
Remember too: These VMs are potentially used across multiple projects.  So for example, `buildah` CI would also need to behave with having both installed.  The safest thing is to download it like this, then manage the install/swap at test-time.  This gets consistent/reliable package versions, and won't mess with other automation.  Where were you thinking of testing this from.  Manually or in a script or in ginkgo or somewhere else?

Reply:
I have confirmed that `containers.conf` will correctly select `runc` as the default on a cgroups v1 system if both runc and crun are available. Buildah does *not* use this bit of containers.conf (which seems like an unfortunate oversight...) but it, too, correctly uses runc on v1 systems even if both are installed.

I think it ought to be safe to have both runtimes installed given this.

Reply:
Thanks guys.  Okay so then for consistency we should probably do the same thing on both Fedora and Ubuntu (install crun & runc).  Once the images are available, I'll open PRs for podman and buildah and some of the other repos.  Both to be on the safe-side, and because many/most projects are using old-ish images.